pkg/eestream/aesgcm_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package eestream
+
+import (
+	"bytes"
+	"io/ioutil"
+	"testing"
+)
+
+func TestAesGcm(t *testing.T) {
```

Comment:
Based on my understanding, the aes_gcm is the first step in getting close to current storj. Next step is to add/port the existing libstorj's aes encrytion/decrytion in to this file, so that the file names and bucket names are encrypted/decrypted per the current libstorj implementation. This  code just give  a functionality plugin to switch to any supported encryptions.

Reply:
:+1: